### ISSUE TITLE
Tor Browser developers key auto update does not work with Flatpak

### ISSUE THREAD

**Issue by AsciiWolf** [flathub/com.github.micahflee.torbrowser-launcher#21]
Tor Browser Launcher has an auto update mechanism for the Tor Browser developers key, however this mechanism does not seem to work with the Flatpak build. This was already discussed [here](https://github.com/flathub/com.github.micahflee.torbrowser-launcher/issues/14), but I am opening a separate issue for it since it is still a problem.

/cc @micahflee

**Comment by rogers0** [comment]
@AsciiWolf 
I tried on my debian buster (with backports) system.
```
$ GNUPGHOME=~/.var/app/com.github.micahflee.torbrowser-launcher/data/torbrowser/gnupg_homedir gpg --auto-key-locate nodefault,wkd --locate-keys torbrowser@torproject.org
gpg: key 4E2C6E8793298290: "Tor Browser Developers (signing key) <torbrowser@torproject.org>" 1 new signature
gpg: Total number processed: 1
gpg:         new signatures: 1
pub   rsa4096 2014-12-15 [C] [expires: 2025-07-21]
      EF6E286DDA85EA2A4BA7DE684E2C6E8793298290
uid           [ unknown] Tor Browser Developers (signing key) <torbrowser@torproject.org>
sub   rsa4096 2018-05-26 [S] [expires: 2022-01-04]
```

so it should be able to refresh the key, by using gnupg out of flatpak.
maybe the gnupg package in flatpak is too old?

**Comment by micahflee** [comment]
When I try inside the flatpak container (which runs gpg 2.2.27) I get a "Invalid URI" error:

```
$ flatpak run --command=sh --devel com.github.micahflee.torbrowser-launcher
[📦 com.github.micahflee.torbrowser-launcher ~]$ gpg --version
gpg (GnuPG) 2.2.27
libgcrypt 1.8.7
Copyright (C) 2021 Free Software Foundation, Inc.
License GNU GPL-3.0-or-later <https://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Home: /home/user/.gnupg
Supported algorithms:
Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
        CAMELLIA128, CAMELLIA192, CAMELLIA256
Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
Compression: Uncompressed, ZIP, ZLIB, BZIP2

[📦 com.github.micahflee.torbrowser-launcher ~]$ gpg -v --status-fd 2 --homedir .var/app/com.github.micahflee.torbrowser-launcher/data/torbrowser/gnupg_homedir --auto-key-locate nodefault,wkd --locate-keys torbrowser@torproject.org
gpg: using pgp trust model
gpg: no running Dirmngr - starting '/usr/bin/dirmngr'
gpg: waiting for the dirmngr to come up ... (5s)
gpg: connection to dirmngr established
gpg: error retrieving 'torbrowser@torproject.org' via WKD: Invalid URI
gpg: error reading key: Invalid URI
```

When I try directly on my computer, right Ubuntu 21.04 (with gpg 2.2.20) it works fine:

```
$ gpg --version
gpg (GnuPG) 2.2.20
libgcrypt 1.8.7
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Home: /home/user/.gnupg
Supported algorithms:
Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
        CAMELLIA128, CAMELLIA192, CAMELLIA256
Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
Compression: Uncompressed, ZIP, ZLIB, BZIP2

$ gpg -v --status-fd 2 --homedir .var/app/com.github.micahflee.torbrowser-launcher/data/torbrowser/gnupg_homedir --auto-key-locate nodefault,wkd --locate-keys torbrowser@torproject.org
gpg: using pgp trust model
gpg: pub  rsa4096/4E2C6E8793298290 2014-12-15  Tor Browser Developers (signing key) <torbrowser@torproject.org>
[GNUPG:] IMPORT_OK 0 EF6E286DDA85EA2A4BA7DE684E2C6E8793298290
gpg: Note: signature key 2D000988589839A3 has been revoked
gpg: key 4E2C6E8793298290: "Tor Browser Developers (signing key) <torbrowser@torproject.org>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
[GNUPG:] IMPORT_RES 1 0 0 0 1 0 0 0 0 0 0 0 0 0 0
gpg: auto-key-locate found fingerprint EF6E286DDA85EA2A4BA7DE684E2C6E8793298290
gpg: Note: signature key 2D000988589839A3 has been revoked
[GNUPG:] KEY_CONSIDERED EF6E286DDA85EA2A4BA7DE684E2C6E8793298290 0
gpg: automatically retrieved 'torbrowser@torproject.org' via WKD
pub   rsa4096 2014-12-15 [C] [expires: 2025-07-21]
      EF6E286DDA85EA2A4BA7DE684E2C6E8793298290
uid           [ unknown] Tor Browser Developers (signing key) <torbrowser@torproject.org>
sub   rsa4096 2014-12-15 [S] [revoked: 2015-08-26]
sub   rsa4096 2018-05-26 [S] [expires: 2022-01-04]
```

Hmm...

**Comment by micahflee** [comment]
I believe this is now fixed in version 0.3.5 :)
